app/views/admin/questions/_level.html.slim

```diff
@@ -0,0 +1,13 @@
+.progress
+  - level = @question.level
```

Comment:
good that now you have one line with progressbar class
but - still let's DRY it
consider using helpers